public/docs/_examples/systemjs.config.plunker.js

```diff
@@ -56,6 +56,9 @@
 
   // No umd for router yet
   packages['@angular/router'] = { main: 'index.js', defaultExtension: 'js' };
+  
+  // Forms not on rc yet
+  packages['@angular/forms'] = { main: 'index.js', defaultExtension: 'js' };
```

Comment:
I think it will also need changes above in the mapping, the same treatment as the router (locking the version and mapping with it).